espnet/nets/pytorch_backend/transducer/decoder_interface.py

```diff
@@ -0,0 +1,109 @@
+"""Transducer decoder interface module."""
+
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
+
+import torch
+
+
+class TransducerDecoderInterface:
+    """Decoder interface for transducer models."""
+
+    def zero_state(
+        self, init_tensor: torch.Tensor = None
+    ) -> Union[Tuple[List[torch.Tensor], List[torch.Tensor]], List[torch.Tensor]]:
```

Comment:
LGTM overall. I think the type of states should be `typing.Any` because it deprends on model architecture. And there is no need to restrict it if beam search implementations don't care about that.

Reply:
Thanks for the tip, done!